bin/test_configure_tower.py

```diff
@@ -0,0 +1,237 @@
+#!/usr/bin/env python3
+
+from copy import deepcopy
+import os
+
+import boto3
+from botocore.stub import Stubber
+import pytest
+
+import configure_tower
```

Comment:
Will the testing work when you run from a different directory?

```
pytest bin/test_configure_tower.py 
```

Since `configure_tower` isn't a package, will pytest try to import a python packge?

Reply:
I plan on reorganizing the files once #62 and #64 are merged. I just don't want to create a headache when I merge changes from `main` down the road. I'm not sure what this will look like yet. Do you have any suggestions?

Reply:
Hmm, I'm actually unsure, I do feel like a lot of stuff here could eventually be superceded by a tower python client.  I think there's many solutions.

* The more complex one: utilize `importlib` package and import it after getting the true path
* The easier one, change the working dir in the test script

```
import os
directory = os.path.dirname(os.path.abspath(__file__))
os.chdir(directory)
```

Reply:
I'll leave this conversation as unresolved as a reminder to tackle this once the other PRs are merged.